### PR TITLE
doc: Update to state nRF91 Series DK instead of nRF9160

### PR DESCRIPTION
**Pull request by divipillai**
with nRF9161 support, Update the documentation to
state nRF9161 or nRF91 series DK instead of nRF9160.

NCSDK-22136

### CODE REVIEW

**Comment by umapraseeda** [comment]
@gmarull, could you check the compliance error: https://github.com/nrfconnect/sdk-nrf/actions/runs/6826096329/job/18565265770?pr=13052? The PR is editing the image that is already present in the repo and the  image size is presently 811 kB (after edits). Is this something that can be ignored?

**Comment by gmarull** [comment]
> @gmarull, could you check the compliance error: https://github.com/nrfconnect/sdk-nrf/actions/runs/6826096329/job/18565265770?pr=13052? The PR is editing the image that is already present in the repo and the image size is presently 811 kB (after edits). Is this something that can be ignored?

just ignore the message. does the SVG contain binary data inside? I'm surprised for such a big SVG file.

**Comment by divipillai** [comment]
> > @gmarull, could you check the compliance error: https://github.com/nrfconnect/sdk-nrf/actions/runs/6826096329/job/18565265770?pr=13052? The PR is editing the image that is already present in the repo and the image size is presently 811 kB (after edits). Is this something that can be ignored?
> 
> just ignore the message. does the SVG contain binary data inside? I'm surprised for such a big SVG file.

I believe we don't have binary data inside. The build error is because of the size.

File: samples/wifi/shell/README.rst [hunk 1]
```diff
@@ -51,12 +51,25 @@ The following is an example of the CLI command:
...
 
-.. code-block:: console
+   .. group-tab:: nRF9161 DK
+
+      To build for the nRF9161 DK, use the ``nrf9161dk_nrf9161_ns`` build target with the ``SHIELD`` CMake option set to ``nrf7002ek`` and a scan-only overlay configuration.
+      The following is an example of the CLI command:
+
+      .. code-block:: console
+
+         west build -p -b nrf9161dk_nrf9161_ns -- -DOVERLAY_CONFIG=overlay-scan-only.conf -DSHIELD=nrf7002ek
```
Comment (krish2718) [inline samples/wifi/shell/README.rst:63]: We could not use `nrf7002ek_nrf7000` itself, but can be done in another PR as well.

**Review by krish2718 — APPROVED** [review]
For Wi-Fi parts.

**Review by junqingzou — APPROVED** [review]
OK with ZZHC

File: samples/cellular/location/README.rst [hunk 2]
```diff
@@ -70,16 +70,24 @@ Building and running
...
 
-For example:
+.. tabs::
 
-.. code-block:: console
+   .. group-tab:: nRF9161 DK
+
+      .. code-block:: console
+
+          west build -p -b nrf9161dk_nrf9161_ns -- -DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf -DSHIELD=nrf7002ek
```
Comment (tokangas) [inline samples/cellular/location/README.rst:84]: The `nrf9160dk_with_nrf7002ek.overlay` overlay file is needed also with nRF9161 DK. The name of the overlay is misleading as it's used with both DKs, we'll update it.
```suggestion
          west build -p -b nrf9161dk_nrf9161_ns -- -DSHIELD=nrf7002ek -DDTC_OVERLAY_FILE=nrf9160dk_with_nrf7002ek.overlay -DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf
```

**Review by tokangas — CHANGES_REQUESTED** [review]

File: samples/cellular/location/README.rst (hunk 2 above)
Comment (tokangas) [reply samples/cellular/location/README.rst:84]: Created PR #13168 which renames the nRF7002 EK overlays in `location` and `modem_shell` samples.

Comment (divipillai) [reply samples/cellular/location/README.rst:84]: I will update these changes in this PR.

File: doc/nrf/test_and_optimize/optimizing/power_nrf91.rst [hunk 3]
```diff
@@ -108,7 +108,7 @@ Assumptions:
 * The network is not present in the drop-down list corresponding to :guilabel:`Preset network parameters` in Online Power Profiler.
 
 .. note::
-   All measurements and calculations in the example use case are based on nRF9160 Revision 2.
+   All measurements and calculations in the example use case are based on nRF91 Series Revision 2.
```
Comment (eivindj-nordic) [inline doc/nrf/test_and_optimize/optimizing/power_nrf91.rst:111]: This is still `nRF9160 Revision 2`
https://devzone.nordicsemi.com/power/w/opp/3/online-power-profiler-for-lte

**Review by tokangas — APPROVED** [review]

**Review by eivindj-nordic — APPROVED** [review]

File: doc/nrf/test_and_optimize/optimizing/power_nrf91.rst [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 
 .. contents::
    :local:
    :depth: 2
 
-In addition to the :ref:`general power optimization recommendations <app_power_opt_general>`, you can use the `Online Power Profiler (OPP)`_ when optimizing the power consumption of your nRF9160 application.
+In addition to the :ref:`general power optimization recommendations <app_power_opt_general>`, you can use the `Online Power Profiler (OPP)`_ when optimizing the power consumption of your cellular application.
 
-Online Power Profiler for LTE is a tool to estimate the current consumption of the nRF9160 LTE modem.
+Online Power Profiler for LTE is a tool to estimate the current consumption of the nRF91 Series LTE modem.
```
Comment (maxd-nordic) [inline doc/nrf/test_and_optimize/optimizing/power_nrf91.rst:12]: This is not wrong, but a bit misleading: OPP only supports nRF9160 at the moment.

**Review by maxd-nordic — APPROVED** [review]

**Review by bjda — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

File: doc/nrf/templates/application_README.rst [hunk 5]
```diff
@@ -9,7 +9,7 @@ Template: Application
 
 .. note::
    * Provide a concise name to the application, which corresponds to the folder name.
-     If the application targets a specific device, add it in the title before the application name (for example, "nRF9160:").
+     If the application targets a specific device, add it in the title before the application name (for example, "nRF5340:").
```
Comment (joerchan) [inline doc/nrf/templates/application_README.rst:12]: Maybe add an 'or' to give an example for the nrf91 series, or is it no longer possible to do this?

**Review by joerchan — APPROVED** [review]

File: doc/nrf/templates/application_README.rst (hunk 5 above)
Comment (divipillai) [reply doc/nrf/templates/application_README.rst:12]: Applications that support RF91 Series devices won't have the prefix as they support more than one device. For example, Asset Tracker v2 application. Hence, we could keep this as is.

**Pull request merged**